tardist

```diff
@@ -31,7 +31,7 @@ mkdir  $distdir
 
 cp -p $(<MANIFEST) $distdir/
 
-tar cvzf $tarball $distdir
+GZIP=-n tar --owner=0 --group=0 --numeric-owner -cvzf $tarball $distdir
```

Comment:
gzip(1) says "This feature will be removed in  a  future  release". How about something like:
```bash
tartmp=$distdir-tmp-$$.tar
tar --owner=0 --group=0 --numeric-owner -cvf $tartmp && gzip -n $tartmp && mv -f $tartmp.gz $tarball
```
(removing the tmp steps if clean code &gt;&gt; paranoia)

Reply:
My man page doesn't have that comment, so I don't know what feature of gzip will be removed.  You are still using -n, so maybe you mean the GZIP envvar?

In any case, in gzip 1.10, the -n behavior became the default when compressing a pipe (as with tar), so we could just omit that part entirely.

Reply:
>My man page doesn't have that comment

A thousand pardons! Yes, the 1.9 and 1.10 man page both deprecate the GZIP envariable.

If you're OK with the warning message[1], I think using the envariable should be safe.

```
[1] gzip: warning: GZIP environment variable is deprecated; use an alias or script
```

Reply:
I have gzip 1.6, so I didn't notice the warning.  And I think the warning is worse than the problem -n is trying to ameliorate.  So let's get rid of -n